src/plugins/recordTypes/audit/columns.js

```diff
@@ -0,0 +1,35 @@
+import { defineMessages } from 'react-intl';
+
+export default (configContext) => {
+  const {
+    formatTimestamp,
+  } = configContext.formatHelpers;
+
+  return {
+    default: {
+      principal: {
+        messages: defineMessages({
+          label: {
+            id: 'column.audit.default.updatedBy',
+            defaultMessage: 'By',
+          },
+        }),
+        order: 10,
+        sortBy: 'audits_common:principal',
+        width: 450,
+      },
+      eventDate: {
+        formatValue: formatTimestamp,
+        messages: defineMessages({
+          label: {
+            id: 'column.audit.default.updatedAt',
+            defaultMessage: 'Updated',
+          },
+        }),
+        order: 30,
+        sortBy: 'audits_common:eventDate',
```

Comment:
The REST API doesn't seem to support sorting audit records, so this can be removed for now. If we decide that these need to be sortable, I think we'll have to do some work in the services layer.